.github/workflows/e2e.yml

```diff
@@ -0,0 +1,67 @@
+name: Spark Operator E2E Test Suites
```

Comment:
Instead of adding this, can we use the existing one?

https://github.com/apache/spark-kubernetes-operator/blob/f5ce844f49ea36e87a257f9ff86bd3478448178d/.github/workflows/build_and_test.yml#L63-L94

Reply:
updated.